npm/webdriverio.json

```diff
@@ -0,0 +1,6 @@
+{
+  "versions": {
+    "4.2.16-s": "github:sfali23/typings-webdriverio/standalone#49ae26a9976863a61619b715e49ec2aeefa44257",
```

Comment:
The NPM package version is `4.2.16` but the there are two way access API, standalone and with wdio test runner, in standalone mode API works asynchronously while in wdio mode API works synchronously.

I exposed API for both modes, in order to differentiate two version I added suffix in the version.